detection-rules/attachment_fake_gmail_attachment.yml

```
@@ -5,8 +5,19 @@ severity: "high"
 source: |
   type.inbound
   and any([body.html.display_text, body.current_thread.text, body.plain.raw],
-          strings.icontains(., 'Scanned by Gmail')
-          and regex.icontains(., '[KM]b\b')
+          (
```

add length limit to avoid matching on very long [messages](https://platform.sublime.security/messages/49090324b32e070eebaa4ea8b70a3d75e68a814820a0b4a35eec47777a894b6c?preview_id=01958b42-5851-75c3-9af3-7c14bd4fd138). 

```suggestion
          length(.) < 2500 and (
          
```